<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorAtemAudioInputBalanceWidget</class>
 <widget class="QWidget" name="InspectorAtemAudioInputBalanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>116</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="labelBalance">
     <property name="text">
      <string>Balance</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayoutTriggerOnNext">
     <item>
      <widget class="QCheckBox" name="checkBoxTriggerOnNext">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerTriggerOnNext2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelTriggerOnNext">
     <property name="text">
      <string>Trigger on next</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelInput">
     <property name="text">
      <string>Input</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxBalance">
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="sliderBalance">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboBoxInput">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxInput</tabstop>
  <tabstop>sliderBalance</tabstop>
  <tabstop>doubleSpinBoxBalance</tabstop>
  <tabstop>checkBoxTriggerOnNext</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>comboBoxInput</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>InspectorAtemAudioInputBalanceWidget</receiver>
   <slot>inputChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBalance</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorAtemAudioInputBalanceWidget</receiver>
   <slot>sliderBalanceChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxBalance</sender>
   <signal>valueChanged(double)</signal>
   <receiver>InspectorAtemAudioInputBalanceWidget</receiver>
   <slot>doubleSpinBoxBalanceChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggerOnNext</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorAtemAudioInputBalanceWidget</receiver>
   <slot>triggerOnNextChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputChanged(int)</slot>
  <slot>sliderBalanceChanged(int)</slot>
  <slot>triggerOnNextChanged(int)</slot>
  <slot>doubleSpinBoxBalanceChanged(double)</slot>
 </slots>
</ui>
